index.js

```diff
@@ -4,30 +4,34 @@ const asana = require('asana');
 
 async function asanaOperations(
   asanaPAT,
-  projectId,
+  targets,
   taskId,
-  sectionName,
   taskComment
 ) {
   try {
     const client = asana.Client.create({
       defaultHeaders: { 'asana-enable': 'new-sections,string_ids' },
       logAsanaChangeWarnings: false
     }).useAccessToken(asanaPAT);
-    if (sectionName) {
-      let project = await client.sections.findByProject(projectId);
-      if (project) {
-        let requiredSection = project.find(data => data.name === sectionName);
-        if (requiredSection) {
-          await client.sections.addTask(requiredSection.gid, { task: taskId });
-          core.info('Moved to: ' + requiredSection.name);
+
+    const task = await client.tasks.findById(taskId);
+    
+    targets.forEach(async target => {
+      let targetProject = task.projects.find(project => project.name === target.project);
+      if (targetProject) {
+        let targetSection = await client.sections.findByProject(targetProject.gid)
+          .then(sections => sections.find(section => section.name === target.section));
+        if (targetSection) {
+          await client.sections.addTask(targetSection.gid, { task: taskId });
+          core.info(`Moved to: ${target.project}/${target.section}`);
         } else {
-          core.error('Asana section ' + sectionName + ' not found.');
+          core.error(`Asana section ${target.section} not found.`);
```

Comment:
are these quotes or backticks? These seem to be present in other places in the code.

Reply:
backticks are how modern javascript does string interpolation